semgrep-core/src/matching/Matching_generic.ml

```diff
@@ -658,21 +658,29 @@ let m_string a b = if a =$= b then return () else fail ()
 
 (*e: function [[Matching_generic.m_string]] *)
 
-(*s: function [[Matching_generic.string_is_prefix]] *)
-let string_is_prefix s1 s2 =
+(*s: function [[Matching_generic.filepath_is_prefix]] *)
```

Comment:
Do not modify those comments. They will prevent me to sync the code with some documentation I have on the side.
I know this function changed names, but just do not care about those comments.